console/src/main/scala/io/shiftleft/console/QueryDatabase.scala

```diff
@@ -45,8 +45,13 @@ class QueryDatabase(defaultArgumentProvider: DefaultArgumentProvider = new Defau
     * */
   def queriesInBundle[T <: QueryBundle](bundle: Class[T]): List[Query] = {
     queryCreatorsInBundle(bundle).map {
-      case (method, args) =>
-        method.apply(args: _*).asInstanceOf[Query]
+      case (method, args) => {
+        val query = method.apply(args: _*).asInstanceOf[Query]
+        val bundleNamespace = classToType(bundle).typeSymbol.fullName.toString
+        // the namespace currently looks like `io.joern.scanners.c.CopyLoops`
+        val language = bundleNamespace.split('.')(3)
```

Comment:
Does this `(3)` throw an exception if the namespace has fewer .-separated elements? If so, maybe we can implement a version here that tries to fetch the fourth element but returns an empty string if that fails.